Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -349,6 +350,16 @@ - (void)updateReminderViews
         || !self.deregisteredView.isHidden || !self.outageView.isHidden;
 }
 
+- (void)setHasVisibleReminders:(BOOL)hasVisibleReminders
+{
+    if (_hasVisibleReminders == hasVisibleReminders) {
+        return;
+    }
+    _hasVisibleReminders = hasVisibleReminders;
+    // If the reminders show/hide, reload the table.
```

Comment:
If the reminders show/hide, reload the table since a cell needs to be added/removed.